rca/people/tests/test_forms.py

```diff
@@ -0,0 +1,96 @@
+                "introduction": "",
+                "bio": "",
+                "related_project_pages-TOTAL_FORMS": "0",
+                "related_project_pages-INITIAL_FORMS": "0",
+                "related_project_pages-MIN_NUM_FORMS": "0",
+                "related_project_pages-MAX_NUM_FORMS": "5",
+                "gallery_slides-TOTAL_FORMS": "0",
+                "gallery_slides-INITIAL_FORMS": "0",
+                "gallery_slides-MIN_NUM_FORMS": "0",
+                "gallery_slides-MAX_NUM_FORMS": "5",
+                "biography": "",
+                "degrees": "",
+                "experience": "",
+                "awards": "",
+                "funding": "",
+                "exhibitions": "",
+                "publications": "",
+                "research_outputs": "",
+                "conferences": "",
+                "additional_information_title": "",
+                "addition_information_content": "",
+                "relatedlinks-TOTAL_FORMS": "0",
+                "relatedlinks-INITIAL_FORMS": "0",
+                "relatedlinks-MIN_NUM_FORMS": "0",
+                "relatedlinks-MAX_NUM_FORMS": "5",
+                "related_area_of_expertise-TOTAL_FORMS": "0",
+                "related_area_of_expertise-INITIAL_FORMS": "0",
+                "related_area_of_expertise-MIN_NUM_FORMS": "0",
+                "related_area_of_expertise-MAX_NUM_FORMS": "1000",
+                "related_research_centre_pages-TOTAL_FORMS": "0",
+                "related_research_centre_pages-INITIAL_FORMS": "0",
+                "related_research_centre_pages-MIN_NUM_FORMS": "0",
+                "related_research_centre_pages-MAX_NUM_FORMS": "1000",
+                "related_schools-TOTAL_FORMS": "0",
+                "related_schools-INITIAL_FORMS": "0",
+                "related_schools-MIN_NUM_FORMS": "0",
+                "related_schools-MAX_NUM_FORMS": "1000",
+                "personal_links-TOTAL_FORMS": "0",
+                "personal_links-INITIAL_FORMS": "0",
+                "personal_links-MIN_NUM_FORMS": "0",
+                "personal_links-MAX_NUM_FORMS": "5",
+                "student_funding": "null",
+                "seo_title": "",
+                "show_in_menus": "on",
+                "search_description": "",
+                "social_image": "",
+                "social_text": "",
+                "listing_image": "",
+                "listing_title": "",
+                "listing_summary": "",
+                "go_live_at": "",
+                "expire_at": "",
+                "student_user_image_collection": self.collection.id,
+                "student_user_account": "",
+            }
+        )
+
+    def get_form(self, instance, data=None):
+        return self.form_class(instance=instance, data=data)
+
+    def test_form_not_valid(self):
+        print(self.get_form_data())
+        form = self.get_form(instance=self.student_page, data=self.get_form_data())
+        self.assertFalse(form.is_valid())
```

Comment:
This isn't passing, the form.is_valid() here should be false, because we aren't setting anything as the student_user_account on line [86](https://github.com/torchbox/rca-wagtail-2019/pull/647/files#diff-8ccc24b43f569e4fc9146e3d79c9e6d2f0e55dfff11c6b60c755d0838769478fR86). It's testing the validation is working [here](https://github.com/torchbox/rca-wagtail-2019/blob/471be5c4d5eca1d1bd5ed2ba81b9304263bff4cf/rca/people/admin_forms.py#L44-L54). In the admin UI this is fine.